src/org/labkey/targetedms/view/passport/ProteinWebPart.jsp

```diff
@@ -0,0 +1,251 @@
+<%@ page import="org.labkey.api.settings.AppProps" %>
+<%@ page import="org.labkey.api.view.ActionURL" %>
+<%@ page import="org.labkey.api.view.HttpView" %>
+<%@ page import="org.labkey.api.view.JspView" %>
+<%@ page import="org.labkey.api.view.ViewContext" %>
+<%@ page import="org.labkey.targetedms.model.passport.IKeyword" %>
+<%@ page import="org.labkey.targetedms.model.passport.IProtein" %>
+<%@ page import="java.text.SimpleDateFormat" %>
+<%@ page import="java.util.ArrayList" %>
+<%@ page import="java.util.List" %>
+<%@ page import="org.labkey.api.view.template.ClientDependencies" %>
+<%@ page import="org.labkey.targetedms.TargetedMSController" %>
+<%@ taglib prefix="labkey" uri="http://www.labkey.org/taglib" %>
+
+<%@ page extends="org.labkey.api.jsp.JspBase" %>
+
+<link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+<script src="//code.jquery.com/ui/1.11.4/jquery-ui.js" type="text/javascript"></script>
```

Comment:
Need to update to our internal jQuery version

Reply:
Or maybe this is already updated in addClientDependencies() and we just need to delete this

Reply:
this is jquery-ui library.

Reply:
Ah, OK. Let's check it into the TargetedMS module instead of referencing it on the third-party server.

Reply:
I know from visiting China that accessing JQuery on a remote server does not work there. Several uses added by Yuval, the author of the Passport module were totally broken there because of this limitation. So, I agree with Josh. Copy the JQuery you want and refer to it locally.